NorwegianRefugeeCouncil/core
# [CORE24-123] task(setup): Created db lib using knex and used it to run migrations on app start

Jira ticket: JIRA-CORE24-123


## What have you implemented?

Created a `db` lib using knex to connect to the database.
On server start, run the database migrations. Only for test and development environments.
For production we should rather make it part of the deployment process, to avoid concurrency issues.

I also setup `testcontainers` library to run e2e tests.

## How to test it?

Serve the app, it should log that migrations were run (since there's no migrations, it actually doesn't run anything, but trying is what matters)

## Extra/next steps

None
